src/FriendsPage.tsx

```diff
@@ -84,17 +84,23 @@ const FriendsPage: React.FC = () => {
         </AutoTable>
       }
     >
-      {!isLoaded ? ("Loading...")
-      : error ? ("An error occured.")
-      : (
+      {!isLoaded ? (
+        "Loading..."
+      ) : error ? (
+        "An error occured."
+      ) : (
         <>
           <AutoTable
             data={friendRequests}
             title={<h3>Friend Requests</h3>}
             hideOnEmpty
           >
             <TableHeader col={"display_name"}>Username</TableHeader>
-            <FriendRequestButtons id={0} onAccept={AcceptFriendship} onReject={RejectFriendship}/>
+            <FriendRequestButtons
+              id={"0"}
```

Comment:
In friend library page, the default is an empty string "", but the default here is "0". It really doesn't matter because it gets overridden anyway, but it may be clearer if it's empty here also